light-zk.js/src/test-utils/createAccounts.ts

```diff
@@ -2,14 +2,14 @@ const solana = require("@solana/web3.js");
 import * as anchor from "@coral-xyz/anchor";
 import { BN } from "@coral-xyz/anchor";
 import {
-  AccountInfo,
   Connection,
   Keypair,
   PublicKey,
   sendAndConfirmTransaction,
 } from "@solana/web3.js";
 const { SystemProgram } = require("@solana/web3.js");
 // const token = require('@solana/spl-token')
+// @ts-ignore
```

Comment:
why is a ts ignore added here?
(if we don't need the import why don't we remove it?)

Reply:
because the "_" underscore drives the linker crazy since crosses the "non-used" variable rule